components/Expire.coffee

```diff
@@ -0,0 +1,40 @@
+noflo = require 'noflo'
+
+# @runtime noflo-nodejs
+
+exports.getComponent = ->
+  c = new noflo.Component
+  c.description = 'Set expiry time for a Redis key'
+  c.icon = 'hourglass'
+  c.inPorts.add 'key',
+    datatype: 'string'
+  c.inPorts.add 'expire',
+    datatype: 'int'
+    description: 'Expiry in seconds'
+  c.inPorts.add 'client',
+    datatype: 'object'
+    description: 'Redis client connection'
+    control: true
+  c.outPorts.add 'out',
+    datatype: 'string'
+  c.outPorts.add 'error',
+    datatype: 'object'
+
+  c.forwardBrackets =
+    key: ['out', 'error']
```

Comment:
What happens if this is not included on components with `out` and `error` ports (extremely common)?

Reply:
@jonnor we provide defaults for `in: ['out', 'error']` but nothing else so far. See noflo/noflo#539

Reply:
Ok, good to know